docs/userguides/decks/micro-sd-card-deck.md

```diff
@@ -20,34 +20,47 @@ The FatFS module will be initialized by the micro-SD deck driver when the deck i
 Without any changes to `usddeck.c` there's a data logging functionality implemented. It can be enabled and configured by placing a config file on the µSD-Card. The file is a simple text file and an example is available [config.txt](https://github.com/bitcraze/crazyflie-firmware/blob/master/tools/usdlog/config.txt). The file format is as follows:
 
 ```
-Frequency
-Number of buffers
-File name
+1     # version
+512   # buffer size in bytes
+log   # file name
+0     # enable on startup (0/1)
+on:fixedFrequency
+100     # frequency in Hz
+1     # mode (0: disabled, 1: synchronous stabilizer, 2: asynchronous)
 log entry 1
 log entry 2
 log entry 3
 ...
+on:myEvent1
+log entry 4
+log entry 5
+...
+on:myEvent2
+...
 ```
 
-Frequency is an integer value in Hertz, for example 250 means that a data block is written every 4ms. The number of buffers is used to decouple the writing to the card and the data logging. Higher frequencies require more buffers, otherwise some data might be lost. Logging 10dof data with 1kHz works well with a buffer of 120 elements. But you may have to try around a bit to get feeling on what is possible. If the buffer was too small or the frequency was too high there will be gaps in the log as you can see in the following picture:
-
-![micro-sd Log Gap](/docs/images/usd_log_gap.png)
+The config file supports logging of [log variables](/docs/userguides/logparam.md) as well as [event triggers](/docs/userguides/eventtrigger.md).
+For the fixed frequency logging, the frequency is an integer value in Hertz, for example 250 means that a data block is written every 4ms. The buffer size is used to decouple the writing to the card and the data logging. Higher frequencies require a larger buffer, otherwise some data might be lost. The Crazyflie console will report after the log file is closed how many events had to be discarded because of insufficient buffer size:
```

Comment:
Maybe _The Crazyflie console will show how many events had to be discarded due to insufficient buffer size._

Reply:
Fixed.